components/atoms/StatusBadge.js

```diff
@@ -25,7 +25,7 @@ export default function StatusBadge(props) {
             : 'bg-status-inactive'
         }`}
       >
-        {t[props.status]}
+        {t[props.status] ? t[props.status] : 'No status found'}
```

Comment:
Just a slightly cleaner way to get the same result (only works because it's not calling a sub-property of the element you are checking)
```suggestion
        {t[props.status] ?? 'No status found'}
```